ext/dapr-ext-workflow/dapr/ext/workflow/workflow_runtime.py

```diff
@@ -80,3 +107,141 @@ def start(self):
     def shutdown(self):
         """Stops the listening for work items on a background thread."""
         self.__worker.stop()
+
+    def workflow(self, __fn: Workflow = None, *, name: Optional[str] = None):
+        """Decorator to register a workflow function.
+
+        This example shows how to register a workflow function with a name:
+
+                from dapr.ext.workflow import WorkflowRuntime
+                wfr = WorkflowRuntime()
+
+                @wfr.workflow(name="add")
+                def add(ctx, x: int, y: int) -> int:
+                    return x + y
+
+        This example shows how to register a workflow function without a name:
+
+                    from dapr.ext.workflow import WorkflowRuntime
+                    wfr = WorkflowRuntime()
+
+                    @wfr.workflow
+                    def add(ctx, x: int, y: int) -> int:
+                        return x + y
+
+        Args:
+            name (Optional[str], optional): Name to identify the workflow function as in
+            the workflow runtime. Defaults to None.
```

Comment:
```suggestion
            the workflow runtime. Defaults to the name of the function.
```

Reply:
In this case, it does default to `None`, but is then overridden by the function name.

Reply:
I guess it depends on whether this description is intended to describe the function signature of the what the user can expect - i.e., the semantic behavior.